app/views/staff/shared/xml/_activity.xml.haml

```diff
@@ -43,6 +43,8 @@
     - benefitting_countries_with_percentages(activity.benefitting_countries).each do |country|
       %recipient-country{"code" => country.code, "percentage" => country.percentage}
         %narrative= country.name
+    %recipient-region{"code" => activity.benefitting_region.code, vocabulary: "1"}
```

Comment:
I think Meyric asked this too, but I didn't want this question to get lost in the massive list of other things he raised:

Are we sure we want to expose the computed region in the XML even where we have country-level precision? I thought the region would only be used for "legacy" activities where the users intentionally gave a `recipient_region`, **or** for activities where we allow users to not select any `benefitting_countries`.

Reply:
Ah, you're right:

> Recipient-region must not be used merely to describe the region of a country reported in recipient-country, but ONLY if the region is a recipient IN ADDITION to the country.